src/HtmlProcessor/CssToAttributeConverter.php

```diff
@@ -20,7 +20,7 @@ class CssToAttributeConverter extends AbstractHtmlProcessor
      * only for certain values, the mapping is an object with a whitelist
      * of nodes and values.
      *
-     * @var mixed[][]
+     * @var array<string, array<string|string[]>>
```

Comment:
Could we be more explicit in the type hint for `$cssToHtmlMap`, e.g. `array<string, array{attribute: string, nodes: ?array<int, string>, values: ?array<int, string>}>`?  (I'm guessing that the nullable operator would work and be needed for optional elements.)

Failing that, the (`string`) key type of the inner array could be specified.

Reply:
Done.